app/src/main/java/com/android/feedme/ui/find/FindRecipeScreen.kt

```diff
@@ -99,6 +99,37 @@ fun FindRecipeScreen(navigationActions: NavigationActions) {
                             ))
                   }
 
+              // Gallery Button
+              OutlinedButton(
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(horizontal = 20.dp)
+                          .padding(bottom = 20.dp)
+                          .testTag("GalleryButton"),
+                  shape = RoundedCornerShape(size = 10.dp),
+                  onClick = { navigationActions.navigateTo(Screen.GALLERY) },
+                  border = BorderStroke(width = 2.dp, color = Color.Black)) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.gallery),
```

Comment:
When defining Icons, you can use directly the materials 3 library for icons by doing for example `Icons.Outlined.Collection` instead of uploading a `painterRessource`, you can find all the available Icons here https://mui.com/material-ui/material-icons/?theme=Outlined&query=galler